source/_integrations/hunterdouglas_powerview.markdown

```diff
@@ -142,13 +148,36 @@ Identify will 'jog' the shade position as a diagnostic tool to ensure the shade
 
 ## Example Automations
 
+### Calling a Powerview Scene
+
+``` yaml
+alias: "blinds closed at night"
+trigger:
+  platform: time
+  at: "18:00:00"
+action:
+  - service: scene.turn_on
+    target:
+      entity_id: scene.10877
+```
+### Force Update Shade Position
+
+This automation is not recommended for battery-powered shades.
+
 ``` yaml
-- alias: "blinds closed at night"
-  trigger:
-    platform: time
-    at: "18:00:00"
-  action:
-    - service: scene.turn_on
-      target:
-        entity_id: scene.10877
+alias: Force Update
+description: 'Update the position of defined shades'
+mode: single
+trigger:
+  - platform: time_pattern
+    hours: '1'
+condition: []
+action:
+  - service: homeassistant.update_entity
+    data: {}
```

Comment:
```suggestion
```

99% sure this isn't needed

Reply:
likely not - was just a straight copy out of an automation i created in thr i